debian/control

```diff
@@ -16,13 +16,15 @@ Build-Depends: debhelper (>= 9),
                python-testtools,
                python-twisted-core (>= 16.0.0),
                python-txjuju,
+               python-txfixtures,
                python-yaml,
                wget
 
 Package: fake-juju
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends},
          juju-mongodb,
+         juju-mongodb3.2,
```

Comment:
OK, as long as we long term settle on one of them I'm happy. Thanks, :bowing_man: